tests/testsuite/profile_config.rs

```diff
@@ -256,11 +256,11 @@ fn profile_config_all_options() {
         )
         .build();
 
-    p.cargo("build --release -v")
+    p.cargo("check --release -v")
```

Comment:
This test involves codegen, so I don't think it should be modified.

I would probably avoid any changes to `profile_config.rs` to be safe, since they involve profile settings.